src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -297,9 +343,9 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
   module_id_to_info_map_[module_id] = module_metadata;
 
   Debug("ModuleLoadFinished emitted new metadata into ", module_id, " ",
-        module_info.assembly.name, " AppDomain ",
-        module_info.assembly.app_domain_id, " ",
-        module_info.assembly.app_domain_name, ". .");
+       module_info.assembly.name, " AppDomain ",
+       module_info.assembly.app_domain_id, " ",
+       module_info.assembly.app_domain_name, ". .");
```

Comment:
While you're in here, can you remove the `". ."` at the end of this log message? Not sure how that got in there...

Reply:
Fixed!